### DISCUSSION TITLE
want  different values on startAxis and EndAxis of ComposedChart

### DISCUSSION THREAD

**Discussion by Aatif-Thinkitive** [patrykandpatrick/vico#373] [Questions]
Here, I want the startAxis to be as it is but on the EndAxis I want values to be 0,10,20,30,40,50
How can I implement this? 

![image](https://github.com/patrykandpatrick/vico/assets/108254342/a7ae2f4a-fd84-480b-a51b-c5d1e946f5e0)

code:

 
    ProvideChartStyle(rememberChartStyle(columnChartColors, lineChartColors)) {
                val defaultColumns = currentChartStyle.columnChart.columns
                val defaultLines = currentChartStyle.lineChart.lines
                val columnChart = columnChart(
                    remember(defaultColumns) {
                        defaultColumns.map { defaultColumn ->
                            LineComponent(

                                Color(48, 126, 199).toArgb(),
                                thicknessDp = 40f,
                                margins = dimensionsOf(-5.dp),
//                        shape = Shapes.roundedCornerShape(columnCornerRadius),

                            )
                        }
                    },
                    maxY = 100f,
                )
                val lineChart = lineChart(
                    remember(defaultLines) {
                        defaultLines.map { defaultLine ->
                            com.patrykandpatrick.vico.core.chart.line.LineChart.LineSpec(
                                lineColor = Color(162, 165, 55).toArgb(),
                                pointConnector = DefaultPointConnector(cubicStrength = 0.1f),
                                lineThicknessDp = 3.5f,
                                point = ShapeComponent(
                                    color = Color(162, 165, 55).toArgb(),
                                    shape = com.patrykandpatrick.vico.core.component.shape.Shapes.pillShape,
                                    strokeWidthDp = 9f
                                ),
                            )
                        }
                    },
                    maxY = 50f,

                )
                Chart(
                    chart = remember(columnChart ,  lineChart) { columnChart + lineChart },
                    chartModelProducer = composedChartEntryModelProducer,

                    endAxis = endAxis(
                        guideline = null,
                        maxLabelCount = 5,
                        tickLength = 5.dp,

                        tick = LineComponent(color = Color(48, 124, 197).toArgb()),
                        axis = LineComponent(color = Color(48, 124, 197).toArgb()),
                    ),
                    startAxis = startAxis(

                        guideline = null, tickLength = 5.dp,
                        maxLabelCount = 5,
                        axis = LineComponent(color = Color(230, 234, 61).toArgb()),
                        tick = LineComponent(color = Color(230, 234, 61).toArgb()),
                    ),
                    bottomAxis = bottomAxis(
                        guideline = null,

                        sizeConstraint = Axis.SizeConstraint.TextWidth("Week 1"),
                        axis = LineComponent(color = Color(255, 255, 255).toArgb()),
                        tick = null,
                    ),

**Comment by patrickmichalik** [comment]
Hello! I see that you’ve closed the discussion, presumably because you’ve found the answer to your query. In case someone else has the same question and finds this discussion, let me point out that you can achieve the effect described here by using `targetVerticalAxisPosition`.
